File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_bchctlrn.cs
// EXPECTED: 15
// Reference emit for BCHCTLRN — BCHCTL00 FUNC-CHEK USING path.
using System;
using System.Collections.Generic;
public static class BchctlrnRef {
  public static void Main() {
    string func = "CHEK";
    var codes = new Dictionary<string, int> {
      ["INIT"] = 0, ["CHEK"] = 15, ["UPDT"] = 25, ["TERM"] = 35
    };
    int rc = codes.TryGetValue(func, out var v) ? v : 99;
    Console.WriteLine(rc.ToString("00"));
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_bchctlrn.py
"""Reference emit for BCHCTLRN — BCHCTL00 FUNC-CHEK USING path → 15."""
# EXPECTED: 15
FUNC = "CHEK"
RC = {"INIT": 0, "CHEK": 15, "UPDT": 25, "TERM": 35}.get(FUNC, 99)
print(f"{RC:02d}")
